### ISSUE TITLE
Wallet app crashes every time on MacOS

### ISSUE THREAD

**Issue by drewmeeks** [litecoincash-project/litecoincash#8]
### Describe the issue
Yesterday, the app worked. I left it running overnight to sync the blockchain but it may have been interrupted. This morning, when I tried to relaunch the app, it crashed. I tried reinstalling, etc. and it still crashes every time. 

### Can you reliably reproduce the issue?
Yes. 
#### If so, please list the steps to reproduce below:
1. Double click on LitecoinCash Core app icon.
2. Loading block index, rewinding blocks, then crashes quickly.
3. 

### Expected behaviour
App should launch and resume syncing the blockchain.

### Actual behaviour
App crashes. 

### Screenshots.
If the issue is related to the GUI, screenshots can be added to this issue via drag & drop.

### What version of litecoincash-core are you using?
version 0.15.0.1

### Machine specs:
- OS: MacOS 10.13.13
- CPU: 2.5 GHz Intel Core i5
- RAM: 16 GB 1333 MHz DDR3
- Disk size: 512 gb
- Disk Type (HD/SDD): sdd

### Any extra information that might be useful in the debugging process.
Debug log here:
https://privatebin.net/?c9a736bba4367389#rqnKJgGsQGigTTefnfPmyJIyzOnbgjTk6SgwJstxuqQ=
password is github

**Comment by laris2** [comment]
Could you provide debug.log from litecoin cash data directory for more info?
~/Library /Application Support/litecoincash

**Comment by drewmeeks** [comment]
yes, I had to hunt a little bit for that but here's a link to the portion logging one of the crash instances. 
https://privatebin.net/?33d0d288e7257ded#84o+Jo1N6g3cFHivflQRiqFhVxRnNqAovseu3aCUpNc=
same password as before

**Comment by drewmeeks** [comment]
renaming the /LitecoinCash folder and relaunching the app seems to have worked and now I put the bootstrap.dat file in the right place so hopefully the sync will go faster this time. thanks for your help.

**Comment by vijayantjain** [comment]
Facing the same issue 
App is crashing continuously on the Mac.

Please guide and provide the steps to resolve.

**Comment by drewmeeks** [comment]
**To view the hidden ~/Library Folder**

1. Open the Finder
2. Go to your Macintosh HD folder (access this from Devices in the left column)
3. Hold down Cmd + Shift + . (dot)
4. All the hidden files will become visible
5. Hold down Cmd + Shift + . (dot) a second time to hide the files again

**The easiest way to find your hidden ~/Library/ folder is to do the following:**

1. Open Finder
2. Hold down Alt (Option) and choose Go from the dropdown menu bar at the top of the screen
3. You will see the ~/Library folder listed below the Home folder

Then what I did was go into Application Support folder and rename the existing LitecoinCash folder to something else, to force the app to make a new set of data files.  This worked for me and I hope it works for you too!

**Comment by vijayantjain** [comment]
Hi,

Thank you for the prompt reply.
I was able to find the directory and changed its name.
Now the system is synchronizing all the data of approx 15 gb on my system.
Unfortunately the wallet key displayed is different than my original key.

Can you please guide how can i get my original wallet key restored in the application??
As a safety measure i have kept the zipped folder of original LiteCoinCash folder with me.

Any pointers of help will be appreciated.
Thank you.

**Comment by laris2** [comment]
Unfortunately the wallet key displayed is different than my original key.

What do you mean by the wallet key?
The Litecoin Cash is using different addressing system so public keys will be different.
You can check the conversion from LTC -> LCC on their website https://litecoinca.sh
